src/components/PlacementResults.js

```diff
@@ -22,8 +22,12 @@ export default class PlacementResults extends React.Component<void, {}, void> {
     if (response && response.placements && response.documents.length > 0) {
       const placements = response.placements;
       const results = [];
+      let markupCount = 0;
       placements.forEach((placement: Placement) => {
-        const key = `${placement.linkUrl ? placement.linkUrl : 'nourl'}-${placement.linkText ? placement.linkText : 'notext'}-${placement.imageUrl ? placement.imageUrl : 'noimg'}`; // eslint-disable-line max-len
+        if (placement.markup) {
+          markupCount += 1;
+        }
+        const key = `${placement.linkUrl ? placement.linkUrl : 'nourl'}-${placement.linkText ? placement.linkText : 'notext'}-${placement.imageUrl ? placement.imageUrl : 'noimg'}-${placement.markup ? `markup${markupCount}` : 'nomarkup'}`; // eslint-disable-line max-len
```

Comment:
This line is (well, was before, too) very long, with all the ternary operators and nested string substitutions, it's not super clear what we're doing.
What would you think about breaking It up into pieces and then concatenating the pieces to get the final key?
E.g.:
```
const urlKey = placement.linkUrl ? placement.linkUrl : 'noUrl';
const textKey = placement.linkText ? placement.linkText : 'noText';
const imageKey = placement.imageUrl ? placement.imageUrl : 'noImage';
const markupKey = placement.markup ? `markup${markupCount}` :  'noMarkup';
const key = `${urlKey}-${textKey}-${imageKey}-${markupKey}`;
```

Reply:
Yes makes sense. I just realized now that es-lint is disabled on this line. That is why it didn't warn me about the long line!!
Will make this change.